zzzz_modular_occulus/code/modules/client/preferences.dm

```diff
@@ -95,6 +95,7 @@
 	character.appearance_flags	-= fuzzy*PIXEL_SCALE
 
 	character.body_markings = body_markings
+	character.species.blood_color = blood_color
```

Comment:
Every subsequent player that spawns in will override the blood color of any existing players in-round who has the same species, as this directly modifies the species datum shared by everyone who selects that particular species.

Reply:
This is correct. I'm working on a fix for it now.

Reply:
There is two possible solutions. One is to try and make the player mobs species mob it's own local datum rather then referencing a global one.
The other option is to move the blood color onto the mob itself, and then change all the blood decal code to reference the mobs blood color instead of the mobs species blood color.

As you can tell with the commits I just made I've opted for the latter. However I realize in the past this had a problem where players would constantly fill their reagent tables with blood that wasn't theirs. So far this has not been the case in my testing but any relevant information on how I can test for this would be helpful.